massgov/mayflower
# DP-9566 Remove raw filter from _meta/_00-head.twig and _meta/_01-foot.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from:

-  _meta/_00-head.twig 
- _meta/_01-foot.twig

Note:  These templates are not used in Drupal.  `mass_theme/templates/layout/html.html.twig` is used instead.

## Related Issue / Ticket

- [DP-9566](https://jira.mass.gov/browse/DP-9566)

## Steps to Test


1. Go to any pages and check for any display and functional (js) issues from the resources in `<head>` and JS after the page content.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
